pkg/namespace/namespace.go

```diff
@@ -28,16 +28,6 @@ func New(version uint8, id []byte) (Namespace, error) {
 	}, nil
 }
 
-// MustNew returns a new namespace with the provided version and id. It panics
-// if the provided version or id are not supported.
-func MustNew(version uint8, id []byte) Namespace {
-	ns, err := New(version, id)
-	if err != nil {
-		panic(err)
-	}
-	return ns
-}
-
```

Comment:
Out of curiosity, why remove this? This would a breaking change. 

If there are no internal or downstream consumers then we can consider removing this but we may want to tackle in a stand-alone issue + PR.

Reply:
yeah, now no internal or downstream consumers, It will be reopened as needed

Reply:
thank you, I'm rollback and will + PR